pkg/cloud/cloud.go

```diff
@@ -0,0 +1,7 @@
+package cloud
+
+const (
+	Aws   = "AWS"
+	Azure = "AZURE"
+	Gcp   = "GCP"
```

Comment:
Are you sure that every usage of these strings uses the uppercase version? I'm thinking that this change might cause some tests to fail where the lowercase equivalent is expected.

Reply:
Actually, this enum struct has everything as capitals before, I am just replacing its name.

But sure, let me triple check before the merge, although the CI tests already passed.

Reply:
Finished checking, it should be ok, I am not replacing usage of` "aws", "Aws" or "AWS"` everywhere with `pcloud.Aws`.

What I did is just replace the package name, on the places where `network.CloudAws` were invoked.